generators/entity-server/templates/src/main/kotlin/package/service/mapper/EntityMapper.kt.ejs

```diff
@@ -71,28 +71,33 @@ _%>
 
 <%_
 // DTO -> entity mapping
-var renMapAnotDto = false;  //Render Mapping Annotation during DTO to Entity conversion?
+var mappedRelsDto = [];
```

Comment:
Can you check this something is not tallying here with 6.1.2?

Reply:
Yes, this was intentional here as we can not have multiple `@Mapping` as in java and have to use `@Mappings(@Mapping(...), @Mapping())` instead and also the fact that a new mapping was added for the remove operation. So in order to calculate the actual mappings and use of commas properly I used an array instead